bclientapi/bclientapi.go

```diff
@@ -48,7 +49,7 @@ func addFileToTransactionList(filename string, fileID string, item string, isNew
 	fileIDMutex.Unlock()
 }
 
-func New(server string, item string, fileroot string, chunkSize int, wait bool) *itemAttributes {
+func New(server string, item string, fileroot string, chunkSize int, wait bool, token string) *itemAttributes {
```

Comment:
I know you prefer the style

https://github.com/ndlib/bendo/blob/master/cmd/bendo/main.go#L50

to a New with explicit parameters, and I considered it, but honestly, I dislike its looks.